modules/http4s/src/smithy4s/http4s/SimpleProtocolBuilder.scala

```diff
@@ -131,11 +131,26 @@ abstract class SimpleProtocolBuilder[P](
       service: smithy4s.Service[Alg],
       impl: FunctorAlgebra[Alg, F],
       errorTransformation: PartialFunction[Throwable, F[Throwable]],
-      middleware: ServerEndpointMiddleware[F]
+      middleware: ServerEndpointMiddleware[F],
+      encodeErrorsBeforeMiddleware: Boolean
   )(implicit
       F: Concurrent[F]
   ) {
 
+    def this(
+        service: smithy4s.Service[Alg],
+        impl: FunctorAlgebra[Alg, F],
+        errorTransformation: PartialFunction[Throwable, F[Throwable]],
+        middleware: ServerEndpointMiddleware[F]
```

Comment:
The constructor of `RouterBuilder` is package-private, I think we don't need this auxiliary one. Just have to add a MiMa exclusion to the build definition if it shouts at you.

Reply:
Thank you, missed this

done - https://github.com/disneystreaming/smithy4s/pull/1669/commits/e542761625ab0302f0ef0debbb2600a6e1ea54ee

`PartialFunctionRouter` and `KleisliRouter` also defined as `private` so I've added a MiMa exclusion for them as well [here](https://github.com/disneystreaming/smithy4s/pull/1669/commits/f1239f0c61dedfec9873812ef8bcd061d8fc7757)